src/Exception.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Wearesho\Pvbki;
+
+/**
+ * Class Exception
+ * @package Wearesho\Pvbki
+ *
+ * Base exception for all exceptions for this package
+ * Use for handling all library exceptions in your own project
+ *
+ * @example try { ... } catch (Pvbki\Exception $exception) { ... }
+ */
+class Exception extends \Exception
```

Comment:
Its more useful to create interface Exception instead of class because you will be able to extend child classes from \InvalidArgument exception etc.